Form-Controls/index.html

```diff
@@ -13,15 +13,44 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+          <fieldset>
+            <legend>Confirm Personal Details</legend>
+            <label for="name">First Name</label><br />
+            <input type="text" id="name" name="name" required minlength="2" placeholder="John"><br />
+  
+            <label for="email">Email</label><br />
+            <input type="email" id="email" name="email" required><br />
+          </fieldset>
+  
+          <fieldset>
+            <legend>Choices for the Item</legend>
+  
+            <label for="chosen_color">Color</label><br />
```

Comment:
🌱 It's a good idea to get in the practice of using kebab-case for html form controls. When using kebab case in HTML, multi-word names (like chosen color), are separated by a hyphen and kept all in lowercase. In your case, changing your label and select controls to use `chosen-color` instead of `chosen_color` follows best practices.
Read more: https://courses.cs.washington.edu/courses/cse154/17au/styleguide/html-css/naming-conventions-html.html#casing-and-chars